org.gnome.design.Contrast.json

```diff
@@ -0,0 +1,60 @@
+{
+    "app-id": "org.gnome.design.Contrast",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "contrast",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=xdg-run/dconf",
```

Comment:
I think there is an effort going on to remove **the old DConf access method** [here](https://github.com/flathub/flathub/issues/1040) for some applications.

I'm not sure if that applies also to this one (in the future), but it looks like :wink:

Reply:
I'm aware of that, waiting for 3.34 and a bug on gtk to be fixed as you can't run inspector without having the dconf holes